src/styles/mixins.scss

```diff
@@ -0,0 +1,70 @@
+@use "./constants" as *;
+
+@mixin body {
+  font-family: "Source Serif Pro", serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 26px;
+  margin: 0;
+  @media screen and (max-width: $absolute-break) {
+    // font-size: 14px;
```

Comment:
please check for available responsive font sizes and if not used please remove the media query @JS-TSB